app/src/main/java/com/example/studentarena/fragments/ComposeFragment.java

```diff
@@ -162,10 +168,11 @@ private void savePost() {
         Post post = new Post();
         post.setDescription(etDescription.getEditText().getText().toString());
         post.setContact(etContactinfo.getEditText().getText().toString());
-        post.setPrice(etPrice.getEditText().getText().toString());
+        // post.setPrice(etPrice.getEditText().getText().toString());
```

Comment:
should this be commented out? don't you need it to set the price?

Reply:
In my search branch I made an edit which causes a conflict here, I changed my price type from string to double. That's why I commented it.